pinery-miso/pom.xml

```diff
@@ -17,7 +17,7 @@
     <mysql.db>lims</mysql.db>
     <mysql.user>tgaclims</mysql.user>
     <mysql.pw>tgaclims</mysql.pw>
-    <pinery.version>2.25.0</pinery.version>
+    <pinery.version>2.25.1-SNAPSHOT</pinery.version>
```

Comment:
Could you explain what this would do?

Reply:
A snapshot is a version that's not yet released. This PR depends on updates to Pinery that haven't yet been released. I built the Pinery changes on my development machine so I could test this. Before merging this PR, there needs to be a release of Pinery, after which I'll update this to the proper release version

Reply:
Fair enough. Thanks!